Dockerfile

```diff
@@ -24,6 +24,14 @@ RUN apt-get update && \
   && apt-get clean \
   && rmdir /var/cache/apt/archives/partial
 
+# Creating the user is required because jenkins runs he container
+# with the same user as the host (with '-u <uid>:<gid>')
+# but without the user existing 'git' fails with 'No user exists for uid ...'.
```

Comment:
I wonder why `git` behaves like this - why does it even care about local system user presence?

Reply:
Good question, no idea but it does...

Reply:
Looking at the description on https://issues.jenkins.io/browse/JENKINS-47026, it seems not only git fails, ssh also (which is probably why git fails).